static/js/recent_topics.js

```diff
@@ -2,6 +2,7 @@ const render_recent_topics_body = require('../templates/recent_topics_table.hbs'
 const render_recent_topic_row = require('../templates/recent_topic_row.hbs');
 const topics = new Map(); // Key is stream-id:topic.
 let initialized = false;
+const MAX_AVATAR = 3;  // Number of avatars to display
```

Comment:
I think we want to show 4 (or 3 + the count block); is that what this does?  Maybe worth making the comment on its own line and explain this more clearly.

Reply:
Ok, Done!